example/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "example",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "run-p type-check build-only",
+    "preview": "vite preview",
+    "build-only": "vite build",
+    "type-check": "vue-tsc --noEmit -p tsconfig.app.json --composite false",
+    "test:e2e": "start-server-and-test preview http://localhost:4173 'cypress run --e2e'",
+    "test:e2e:dev": "start-server-and-test 'vite dev --port 4173' http://localhost:4173 'cypress open --e2e'"
+  },
+  "dependencies": {
+    "@headlessui/vue": "^1.7.14",
+    "@rotki/ui-library": "../",
+    "pinia": "2.1.3",
+    "tailwindcss": "3.3.2",
+    "vue": "3.3.4",
+    "vue-router": "4.2.2"
+  },
+  "devDependencies": {
+    "@tsconfig/node18": "2.0.1",
+    "@types/node": "18.16.8",
+    "@vitejs/plugin-vue": "4.2.3",
+    "@vue/tsconfig": "0.4.0",
+    "autoprefixer": "10.4.14",
+    "cypress": "12.13.0",
+    "npm-run-all": "4.1.5",
```

Comment:
Out of curiosity what command does this provide?

Reply:
haha, i'll remove it, but it enables running multiple commands

like `pnpm build && pnpm start` as `run-p build start`

Reply:
No worries it's ok, it was just not obvious. no need to remove it if it is used.